docs/content/deploying-to-hippo.md

```diff
@@ -30,72 +32,57 @@ coming soon.
-$ #     run_servers.sh script
-$ git checkout 64cf9334528f1975d7cbff207997d83cee4f19c2
 $ ./run_servers.sh
 ```
 
-Clone the Hippo repo locally. The Hippo commit this doc was tested at is `2a659c11ffc260a552f71fb3dba791916d4e064a`. Set the `BINDLE_URL` environment variable from the `run_servers.sh` script. Run the `dotnet clean && dotnet build` command to build the project. Change into the `src/Web` directory and use `dotnet run` to run the project.
-
-```
-$ git clone git@github.com:deislabs/hippo.git
-$ cd hippo
-$ export BINDLE_URL=http://bindle.local.fermyon.link:8088/v1
-$ dotnet clean && dotnet build
-$ cd src/Web
-$ dotnet run \
-  --Scheduler:Driver=nomad \
-  --Bindle:Url="${BINDLE_URL}"
-```
-
-Check that the Hippo platform is running by making sure you see the hippo dashboard in the browser at https://localhost:5309.
+Check that the Hippo platform is running by making sure you see the hippo
+dashboard in the browser at <http://hippo.local.fermyon.link>
 
 Register an account in Hippo using the Hippo CLI.
 
 ```
-hippo auth register --username user --password PassW0rd! --url https://localhost:5309 -k
+$ hippo register --username user --password PassW0rd! --url http://hippo.local.fermyon.link
 ```
 
 ### Deploy the example http-rust app
 
-Clone the [Spin repo](https://github.com/fermyon/spin) to ensure you have the examples locally. Make sure that the http-rust example is built before deploying.
+Clone the [Spin repo](https://github.com/fermyon/spin) to ensure you have the
+examples locally. Make sure that the http-rust example is built before
+deploying.
 
 ```
 $ git clone git@github.com:fermyon/spin.git
 $ cd spin/examples/http-rust
 $ spin build
 ```
 
-Then, set the relevant environment variables and use `spin deploy` to deploy the app to Hippo. _Note: you can also use flags to configure spin deploy_
+Then, set the relevant environment variables and use `spin deploy` to deploy
+the app to Hippo. _Note: you can also use flags to configure spin deploy_
 
 ```
-$ export HIPPO_URL=https://localhost:5309
-$ export BINDLE_URL=http://bindle.local.fermyon.link:8088/v1
+$ export HIPPO_URL=http://hippo.local.fermyon.link
+$ export BINDLE_URL=http://bindle.local.fermyon.link/v1
 $ export HIPPO_USERNAME=user
 $ export HIPPO_PASSWORD=PassW0rd!
-
-$ spin deploy -k
-
+$ spin deploy
 ```
 
-The spin deploy command packages the application and pushes it to the bindle registry, creates a new app in Hippo and a new channel that the runs the application.
+The `spin deploy` command packages the application and pushes it to the bindle
+registry, creates a new app in Hippo and a new channel that the runs the
+application.
 
 ### Test the app
 
-In the future, spin deploy will give the user a domain to hit for the running app. In the meantime, find the IP address and port for the running app int he Nomad dashboard.
-
-Check the Nomad UI for the running job.
+In the future, `spin deploy` will give the user a domain to hit for the running
```

Comment:
quick thought: how far away are we from implementing this? Will it be in Spin 0.3? @michelleN

Reply:
Right now it does give the domain for the application. @itowlson is working on making sure we also get the available routes in #568

Reply:
This is done.